server/schedule/region_scatterer_test.go

```diff
@@ -197,16 +198,16 @@ func (s *testScatterRegionSuite) scatterSpecial(c *C, numOrdinaryStores, numSpec
 
 	// Each store should have the same number of peers.
 	for _, count := range countOrdinaryPeers {
-		c.Assert(float64(count), LessEqual, 1.1*float64(numRegions*3)/float64(numOrdinaryStores))
-		c.Assert(float64(count), GreaterEqual, 0.9*float64(numRegions*3)/float64(numOrdinaryStores))
+		c.Assert(float64(count), LessEqual, 1.25*float64(numRegions*3)/float64(numOrdinaryStores))
```

Comment:
I think we need to do more tests in the real environment to evaluate the drawback of this change.